### PR TITLE
Close.io integration

### PR DESCRIPTION
**Pull request by SeyZ**

### CODE REVIEW

File: integrations/close.io/setup.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  schema.fields.push({
+    field: 'closeio_lead',
+    type: 'String',
+    reference: collectionName + '_closeio_leads.id',
+    column: null,
+    isSearchable: false,
+    integration: 'close.io'
+  });
+
```
Comment (arnaudbesnier) [inline integrations/close.io/setup.js]: Not integrated in the same way the `refund` action is integrated with Stripe.
Don't know if there is a specific reason.

Not sure that `createField` is the right spot anyway.

File: routes/closeio.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+'use strict';
```
Comment (arnaudbesnier) [inline routes/closeio.js]: I think this file is not used anymore; moved in the `integration` directory?

File: services/closeio-customer-lead-getter.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+'use strict';
+
```
Comment (arnaudbesnier) [inline services/closeio-customer-lead-getter.js]: To Remove?

File: services/closeio-lead-emails-getter.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+'use strict';
```
Comment (arnaudbesnier) [inline services/closeio-lead-emails-getter.js]: To Remove?

File: services/closeio-lead-getter.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+'use strict';
```
Comment (arnaudbesnier) [inline services/closeio-lead-getter.js]: To Remove?

File: services/closeio-leads-getter.js [hunk 6]
```diff
@@ -0,0 +1,70 @@
+'use strict';
```
Comment (arnaudbesnier) [inline services/closeio-leads-getter.js]: To Remove?

File: integrations/close.io/setup.js (hunk 1 above)
Comment (SeyZ) [inline integrations/close.io/setup.js]: • Agreed with `createField` name.
• Yes, it's not the same as `refund` action on Stripe. The `Create Close.io lead` is added on an existing collection (`mapping`). The `refund` action is added on a smart collection (`stripe.payments`)

**Pull request merged**